frontend/src/components/common/Button/index.tsx

```diff
@@ -15,6 +15,7 @@ import {
 import { css, SerializedStyles } from "@emotion/react";
 
 type ButtonProps = RawButtonProps & CommonThemeProps;
+type IconButtonProps = ButtonProps & { open?: boolean };
```

Comment:
Added `sdsStyle` `icon` to common Button component - to add `open` state (and therefore "open" color) to button as per [mocks](https://www.figma.com/design/jEv0ZWQd0Q53aFl9nNRbBy/Portal?node-id=1250-8974&t=lBHkJBLVfa6QxB5Y-4). 

This style is used by both Collection view "more" icon button, and Dataset Row "more" icon button (refer [here](https://github.com/chanzuckerberg/single-cell-data-portal/pull/7234/files#diff-d96d3222524eb4739bb176f9768400695b12eb37e0315bdfd1c2236911d46ff1R7) and [here](https://github.com/chanzuckerberg/single-cell-data-portal/pull/7234/files#diff-9e5d5cc8828db3bdc41c98a2435da2bcc3e74cd021c12ca869ccafc3a9460a8b)).